Issue #49: Create eight-bit divider

AndjelaS997 opened the pull request:
- Added 8-bit unsigned divider and self-checking testbench:
- eight_bit_divider: Combinational 8-bit unsigned divider using shift–compare–subtract algorithm with explicit handling of division by zero
- eight_bit_divider_tb: Self-checking testbench that verifies quotient and remainder for several test cases, including division by zero behavior

AndjelaS997 committed:
Issue #49: Create eight-bit divider

- Added 8-bit unsigned divider and self-checking testbench:
- eight_bit_divider: Combinational 8-bit unsigned divider using shift–compare–subtract algorithm with explicit handling of division by zero
- eight_bit_divider_tb: Self-checking testbench that verifies quotient and remainder for several test cases, including division by zero behavior

AndjelaS997 committed:
Issue #49: Create eight-bit divider

- Added 8-bit unsigned divider and self-checking testbench:
- eight_bit_divider: Combinational 8-bit unsigned divider using shift–compare–subtract algorithm with explicit handling of division by zero
- eight_bit_divider_tb: Self-checking testbench that verifies quotient and remainder for several test cases, including division by zero behavior
- Corrected errors which got on pull request

AndjelaS997 committed:
Issue #49: Create eight-bit divider

- Added 8-bit unsigned divider and self-checking testbench:
- eight_bit_divider: Combinational 8-bit unsigned divider using shift–compare–subtract algorithm with explicit handling of division by zero
- eight_bit_divider_tb: Self-checking testbench that verifies quotient and remainder for several test cases, including division by zero behavior
- Corrected errors which got on pull request

knezicm reviewed (changes requested):
Rezultate simulacije trebate okaciti na task (uz komentar) a ne u folder u repozitorijumu. Uklonite fajl `sim_log.txt` iz ovog foldera i okacite ga na task.

Osim toga, imate neka upozorenja pri simulaciji, pa ne bi bilo lose da malo istrazite zasto se to pojavljuje (pogledajte primjer dat ispod).

```
# ** Warning: NUMERIC_STD."=": metavalue detected, returning FALSE
#    Time: 0 ps  Iteration: 0  Instance: /eight_bit_divider_tb/uut
```

AndjelaS997 committed:
Issue #49: Create eight-bit divider

- Added 8-bit unsigned divider and self-checking testbench:
- eight_bit_divider: Combinational 8-bit unsigned divider using shift–compare–subtract algorithm with explicit handling of division by zero
- eight_bit_divider_tb: Self-checking testbench that verifies quotient and remainder for several test cases, including division by zero behavior
- Corrected eight_bit_divider_tb warnings which got on simulation
- Deleted log file
